cumulusci.yml

```diff
@@ -637,35 +637,32 @@ flows:
         group: NPSP
         steps:
             1:
-                task: enable_pilot_in_scratch_org
-                # Remove the "enable pilot" task after RD2 goes GA
-            2:
                 task: execute_anon
                 # Enables RD2 in Custom Settings
                 options:
                     apex: >
                         npe03__Recurring_Donations_Settings__c rdSettings = npe03__Recurring_Donations_Settings__c.getOrgDefaults();
                         rdSettings.%%%NAMESPACE%%%isRecurringDonations2Enabled__c = true;
                         upsert rdSettings;
-            3:
+            2:
```

Comment:
@RadSFDeveloper You don't need to include these changes since they're already in that other PR (along with other changes).

Reply:
Should I rollback the change or leave it and let the merge into the master align with it?

Reply:
Good question actually. It probably all depends on QA timing. So, let's leave it here for now in case testing starts before #5301 is merged. Worse case there will be a small merge conflict to resolve in this PR for the yml file.